backend/api/tests/test_placeapi.py

```diff
@@ -0,0 +1,48 @@
+import unittest
+
+from api.placeapi import PlacesApiRequest
+
+
+class TestConnectionToApi(unittest.TestCase):
```

Comment:
That's cool that you've added test!
Sorry, I didn't mention it but we will probably use pytest for unit tests.
Feel free to leave it as it is, maybe someone else will rewrite it ;).